tc/sgx/trusted_worker_manager/enclave_untrusted/enclave_bridge/signup.h

```diff
@@ -24,19 +24,33 @@ namespace tcf {
         namespace enclave_data {
             // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
             tcf_err_t CreateEnclaveData(
```

Comment:
We need 3 files signup_singleton.h, signup_kme.h, signup_wpe.h, . This will help us to generate independent binaries for each.

Reply:
This would result in 3 different enclave_bridge (untrusted libararies) and this would eventually require 3 different Make files for different enclaves. If this is expected, i will create separate files.

Reply:
Please create separate files. We can decide to use single or different bridge library later. In this PR, include all enclave bridge files in single library.

Reply:
Ok